### DISCUSSION TITLE
想用PaoPaoGateWay搭配coredns，是否可以使用FakeIP功能呢，这样就可以支持github这种http2.0的网站

### DISCUSSION THREAD

**Discussion by github68616208** [kkkgo/PaoPaoGateWay#150] [问和答]
是否可以使用coredns上面手动增加出国的list解析，实现流量出国呢？
比如 PaoPaoGateWay的网关ip是 192.168.222.254，由于PaoPaoGateWay自带dns服务
我在coredns上配置解析
github.com {
    forward . 192.168.222.254
    log
    cache 600
}
这样是否可以得到网关的FakeIP解析呢？

**Comment by kkkgo** [comment]
可以得到网关的FakeIP解析

**Reply by github68616208** [reply]
好的，多谢大佬，因为是公司公共使用，只用于访问github google chatgpt docker-hub等开发类网站，避免同事看翻墙看视频浪费iplc机器的流量

**Reply by kkkgo** [reply]
保险一点你可以往yaml写规则，默认丢弃，毕竟本地计算机手动hosts写静态路由的话也可能绕过。
